system/api-server/src/file-upload/file-upload.service.ts

```diff
@@ -15,7 +15,7 @@ AWS.config.update({
   secretAccessKey: SECRET_ACCESS_KEY,
 });
 
-@Injectable()
+@Injectable() 
```

Comment:
This is a new type of behavior that I haven't seen in other languages. Can you explain to me what this does and what the benefit is?